integrations/gen_docs_integrations.py

```diff
@@ -0,0 +1,276 @@
+            # build the markdown string
+            md = \
+                f"""<!--startmeta
+meta_yaml: "{meta_yaml}"
+sidebar_label: "{sidebar_label}"
+learn_status: "Published"
+learn_rel_path: "{learn_rel_path}"
+message: "DO NOT EDIT THIS FILE DIRECTLY, IT IS GENERATED BY THE COLLECTOR'S metadata.yaml FILE"
+endmeta-->
+
+{integration['overview']}
+"""
+
+            if integration['metrics']:
+                md += f"""
+{integration['metrics']}
+"""
+
+            if integration['alerts']:
+                md += f"""
+{integration['alerts']}
+"""
+
+            if integration['setup']:
+                md += f"""
+{integration['setup']}
+"""
+
+            if integration['troubleshooting']:
+                md += f"""
+{integration['troubleshooting']}
+"""
+
+            path = meta_yaml.replace("https://github.com/netdata/", "") \
+                .split("/", 1)[1] \
+                .replace("edit/master/", "") \
+                .replace("/metadata.yaml", "")
+
+            # Only if the path exists, this caters for running the same script on both the go and netdata repos.
+            if os.path.exists(path):
+                try:
+                    if not os.path.exists(f'{path}/integrations'):
+                        os.mkdir(f'{path}/integrations')
+
+                    with open(f'{path}/integrations/{sidebar_label.lower().replace(" ", "_").replace("/", "-")}.md', 'w+') as txt:
+                        # add custom_edit_url as the md file, so we can have uniqueness in the ingest script
+                        # afterwards the ingest will replace this metadata with meta_yaml
+                        md = md.replace(
+                            "<!--startmeta", f'<!--startmeta\ncustom_edit_url: \"{meta_yaml.replace("/metadata.yaml", "")}/integrations/{sidebar_label.lower().replace(" ", "_").replace("/", "-")}.md\"')
+
+                        clean_and_write(md, txt)
+                except Exception as e:
+                    print("Error in writing to the collector file", e, integration['id'])
+
+                # If we only created one file inside a collector, add the entry to the symlink_dict, so we can make the link
+                if len(os.listdir(f'{path}/integrations')) == 1:
+                    symlink_dict.update(
+                        {path: f'integrations/{sidebar_label.lower().replace(" ", "_").replace("/", "-")}.md'})
+                else:
+                    try:
+                        symlink_dict.pop(path)
+                    except Exception as e:
+                        # We don't need to print something here.
+                        pass
+
+        except Exception as e:
+            print("Exception in collector md construction", e, integration['id'])
+
+    # kind of specific if clause, so we can avoid running excessive code in the go repo
+    elif integration['integration_type'] == "exporter" and not "go.d.plugin" in os.getcwd():
```

Comment:
```suggestion
    elif integration['integration_type'] == "exporter" and "go.d.plugin" not in os.getcwd():
```

This one is mostly just convention, but it is actually a tiny bit faster as `x not in y` than as `not x in y` (one less operation involved in the bytecode).